packages/calypso-build/webpack.config.js

```diff
@@ -71,6 +71,12 @@ function getWebpackConfig(
 		babelConfig = undefined;
 	}
 
+	let postCssConfigPath = process.cwd();
+	if ( ! fs.existsSync( path.join( postCssConfigPath, 'postcss.config.js' ) ) ) {
+		// Default to this package's PostCSS config
+		postCssConfigPath = __dirname;
+	}
```

Comment:
If you leave the `postCssConfigPath` undefined here, then `postcss-loader` will find the config file in its default location, won't it?

The default location is probably "in the same directory as `webpack.config.js`".

This will stop working if we move the `postcss.config.js` to `sass/postcss.config.js`, as I suggest in my other comment 🙂 Then `postCssConfigPath = './sass';` should work.

Reply:
> If you leave the `postCssConfigPath` undefined here, then `postcss-loader` will find the config file in its default location, won't it?

Yep, I think so. We're actually doing something like that for the babel config [a few lines further up](https://github.com/Automattic/wp-calypso/blob/7a493595bdb8e61fa19ec1e2f6ee41a202139f0d/packages/calypso-build/webpack.config.js#L71) (though that case is a bit different, since we're falling back to some Babel presets there).

I had the choice between being explicit, or implicitly falling back to the tool's default, and chose the former, but don't feel very strongly; happy to go with the latter.

> The default location is probably "in the same directory as `webpack.config.js`".

Yep, I think so :+1: 

> This will stop working if we move the `postcss.config.js` to `sass/postcss.config.js`, as I suggest in my other comment  Then `postCssConfigPath = './sass';` should work.

Yep; but see my [reply above](https://github.com/Automattic/wp-calypso/pull/37060#discussion_r338998031).

Reply:
Alright, let's keep the `postcss.config.js` logic as it is. I wanted to start a discussion rather than stating any strong opinion about it.

Reply:
> I think the conditional is more natural when reverted:
> [...]

True; the way I wrote it was meant to reflect the way the Babel config check is done right above. I'm okay with changing it to your suggestion though.

Reply:
I tried this:

```js
diff --git a/packages/calypso-build/webpack.config.js b/packages/calypso-build/webpack.config.js
index aa140ee263..49c8b140ab 100644
--- a/packages/calypso-build/webpack.config.js
+++ b/packages/calypso-build/webpack.config.js
@@ -71,10 +71,10 @@ function getWebpackConfig(
                babelConfig = undefined;
        }
 
-       let postCssConfigPath = process.cwd();
-       if ( ! fs.existsSync( path.join( postCssConfigPath, 'postcss.config.js' ) ) ) {
-               // Default to this package's PostCSS config
-               postCssConfigPath = undefined;
+       let postCssConfigPath; // undefined means default to this package's PostCSS config
+       // path.resolve() will look in current working dir
+       if ( ! fs.existsSync( path.resolve( 'postcss.config.js' ) ) ) {
+               postCssConfigPath = process.cwd();
        }
 
        const webpackConfig = {

```

But TBH I'm not too happy with all the extra comments that are (IMO) needed now (personally, I wasn't aware of `path.resolve()`'s fallback behavior), so I'll leave as-is :grimacing: